src/views/standalone/network/ReverseProxyView.vue

```diff
@@ -0,0 +1,160 @@
+<script setup lang="ts">
+import { useI18n } from 'vue-i18n'
+import {
+  NeTitle,
+  getAxiosErrorMessage,
+  NeButton,
+  NeEmptyState,
+  NeInlineNotification,
+  NeSkeleton
+} from '@nethserver/vue-tailwind-lib'
+import { onMounted, ref } from 'vue'
+import { useUciPendingChangesStore } from '@/stores/standalone/uciPendingChanges'
+import { ubusCall } from '@/lib/standalone/ubus'
+import ProxyTable from '@/components/standalone/reverse_proxy/ProxyTable.vue'
+import DeleteProxyModal from '@/components/standalone/reverse_proxy/DeleteProxyModal.vue'
+import CreateOrEditProxyDrawer from '@/components/standalone/reverse_proxy/CreateOrEditProxyDrawer.vue'
+
+export type ReverseProxy = {
+  id: string
+  type: string
+  description: string
+  location: string
+  destination: string
+  allow?: string[]
+  certificate?: string
+  domain?: string
+  path?: string
+}
+
+const { t } = useI18n()
+
+const uciChangesStore = useUciPendingChangesStore()
+
+const loading = ref(true)
+const proxies = ref<ReverseProxy[]>([])
+const selectedProxy = ref<ReverseProxy | null>(null)
+const showCreateEditDrawer = ref(false)
+const showDeleteModal = ref(false)
+
+const error = ref({
+  notificationTitle: '',
+  notificationDescription: '',
+  notificationDetails: ''
+})
+
+async function fetchProxies() {
+  try {
+    loading.value = true
+    proxies.value = (await ubusCall('ns.reverseproxy', 'list-proxies')).data.data
+    loading.value = false
+  } catch (err: any) {
+    error.value.notificationTitle = t('error.cannot_retrieve_proxies')
+    error.value.notificationDescription = t(getAxiosErrorMessage(err))
+    error.value.notificationDetails = err.toString()
+  }
```

Comment:
I've fixed this, I've also hidden the contents of the page itself if an error occurs (I don't think it would make sense to show the page in an incorrect state otherwise)